python/mxnet/ndarray/numpy/_op.py

```diff
@@ -600,9 +613,13 @@ def identity(n, dtype=None, ctx=None):
     if n < 0:
         raise ValueError("Input 'n' cannot be negative")
     if ctx is None:
-        ctx = current_context()
-    dtype = _np.float32 if dtype is None else dtype
-    return _npi.identity(shape=(n, n), ctx=ctx, dtype=dtype)
+        ctx = str(current_context())
+    else:
+        ctx = str(ctx)
```

Comment:
is `str` required here?

Reply:
Refer to other similiar works, I think `str` is required here.